cmd/hope/utils/nodes.go

```diff
@@ -15,6 +15,18 @@ import (
 	"github.com/Eagerod/hope/pkg/kubeutil"
 )
 
+type ErrNodeNotFound struct {
```

Comment:
Should probably be `NodeNotFoundError`. `ErrXyz` is more for instances than types I think.